### DISCUSSION TITLE
Postal 2.1 not creating ssl certificates for Tracking Domains

### DISCUSSION THREAD

**Discussion by RKVodde** [postalserver/postal#1821] [Help with using Postal]
Postal 2.1 not creating ssl certificates for Tracking Domains.

After creating a Tracking Domain with ssl enabled it shows checked but still in grey color not green.

And the open and click tracking is not working with the mails.

If we do cURL for tracking domain it shows ssl errors.

**Comment by willpower232** [comment]
Postal 2 removed the necessity for Postal to generate its own SSL certificates for tracking. That icon must be accidentally left behind.

You need to configure your web server to point at Postal with a certificate you make yourself.

Have a look at the information here https://docs.postalserver.io/features/click-and-open-tracking
